packages/KIcon/icons/icn-brain.svg

```diff
@@ -1,8 +1,8 @@
 <svg width="32" height="32" viewBox="-2 -2 28 28" xmlns="http://www.w3.org/2000/svg">
-  <path fill="#A4BBCB" d="M11 4h2v16h-2z"/>
-  <path fill="#A4BBCB" d="M6 8h12v2H6zM2 14h20v2H2zM6 20h12v2H6z"/>
-  <path d="M12 4a2 2 0 100-4 2 2 0 000 4z" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M0,14.75a1.5,1.5 0 1,0 3,0a1.5,1.5 0 1,0 -3,0" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M21,14.75a1.5,1.5 0 1,0 3,0a1.5,1.5 0 1,0 -3,0" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M5.25 10.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM18.75 10.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM5.25 22.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM18.75 22.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3z" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
+  <path fill="#A3BBCC" d="M11 4h2v16h-2z"/>
```

Comment:
Can you add a title tag while we're here?

Reply:
@chrisvdp so I'm going through all the SVGs -- should all have a title? I know a11y the answer is yes, but, for something like `icon-check` would the title just be "Checkmark" or something better?

Reply:
I think at least a basic title is a good backstop for when consumers don't add their own.

Reply:
Comment for transparency: titles are auto-generated within `KIcon` if they do not exist.